futurenet/core/etc/stellar-captive-core.cfg

```diff
@@ -0,0 +1,16 @@
+PEER_PORT=11725
```

Comment:
any reason for this override? It should be 11625 (or specify that port in the address field below)

Reply:
ok, I'll remove, it was a c&p from some sample captive core cfg I used as reference from horizon integration tests.